spring-auto-restdocs-core/src/test/java/capital/scalable/restdocs/payload/JacksonResponseFieldSnippetTest.java

```diff
@@ -56,7 +57,6 @@
 
 public class JacksonResponseFieldSnippetTest extends AbstractSnippetTests {
 
-    private static final String LINE_SEPERATOR = System.lineSeparator();
```

Comment:
Are you intentionally removing the system-dependent `LINE_SEPERATOR`? I would suggest to keep it and move to a single `LINE_SEPERATOR` constant instead of having several ones across the tests.

Reply:
I replaced this ugly looking constant with `fixLineSeparator()` used in tests.